Classes/Emogrifier.php

```diff
@@ -1303,9 +1303,9 @@ private function matchIdAttributes(array $match)
      */
     private function matchClassAttributes(array $match)
     {
-        return ($match[1] !== '' ? $match[1] : '*') . '[contains(concat(" ",@class," "),concat(" ","' .
+        return ($match[1] !== '' ? $match[1] : '*') . '[contains(concat(" ",normalize-space(@class)," "),concat(" ","' .
             implode(
-                '"," "))][contains(concat(" ",@class," "),concat(" ","',
+                '"," "))][contains(concat(" ",normalize-space(@class)," "),concat(" ","',
                 explode('.', substr($match[2], 1))
             ) . '"," "))]';
```

Comment:
Then we'll need a test that breaks without that change. (Otherwise, the change is not needed, or our tests don't test enough.)